theano/sandbox/gpuarray/conv.py

```diff
@@ -202,22 +163,8 @@ def flops(self, inputs, outputs):
     def make_thunk(self, node, storage_map, compute_map, no_recycling):
         node_ = copy.copy(node)
```

Comment:
Because this sets max_threads_dim0 which is then use in c_code() or some other place.